backend/FwLite/MiniLcm/SyncHelpers/EntrySync.cs

```diff
@@ -97,15 +97,27 @@ private static async Task<int> SensesSync(Guid entryId,
 
     private class EntriesDiffApi(IMiniLcmApi api) : ObjectWithIdCollectionDiffApi<Entry>
     {
-        public override async Task<(int, Entry)> AddAndGet(Entry afterEntry)
+        public override async Task<(int, Entry)> AddWithoutReferencesAndGet(Entry afterEntry)
         {
-            //create each entry without components.
+            //create each entry (and its hierarchy: senses, example sentence etc.) in isolation (e.g. without components)
             //After each entry is created, then replace will be called to create those components
             var entryWithoutEntryRefs = afterEntry.WithoutEntryRefs();
             var changes = await Add(entryWithoutEntryRefs);
             return (changes, entryWithoutEntryRefs);
         }
 
+        public override async Task<(int, Entry)> ReplaceWithoutReferencesAndGet(Entry beforeEntry, Entry afterEntry)
+        {
+            //same as AddAndGet, but for already existing entries, because they
+            //might have new entities (e.g. senses) in their hierarchy that other entries reference
+            var beforeEntryWithoutEntryRefs = beforeEntry.WithoutEntryRefs();
```

Comment:
@hahn-kev there's a decent amount of object churn in here. What do you think? It's nice having everything be so generic, but we could make it more specific and explicit and only sync senses.

Reply:
yeah I think it would make more sense to just be specific, it's hard to understand what's going on here right now.

Reply:
also the method `WithoutEntryRefs` is kinda a lie, it includes Sense refs for example.